src/sql/azdata.d.ts

```diff
@@ -2763,12 +2763,12 @@ declare module 'azdata' {
 		withProps(properties: TPropertyBag): ComponentBuilder<TComponent, TPropertyBag>;
 		withValidation(validation: (component: TComponent) => boolean | Thenable<boolean>): ComponentBuilder<TComponent, TPropertyBag>;
 	}
-	export interface ContainerBuilder<TComponent extends Component, TLayout, TItemLayout, TPropertyBag extends ComponentProperties> extends ComponentBuilder<TComponent, TPropertyBag> {
+	export interface ContainerBuilder<TComponent extends Component, TLayout, TItemLayout, TPropertyBag extends ContainerProperties> extends ComponentBuilder<TComponent, TPropertyBag> {
```

Comment:
You can't reference ContainerProperties since it's defined in azdata.proposed - this will break compilation for any extensions trying to use it that don't have azdata.proposed.

It might be ok to just put ContainerProperties in here directly in this case - let me think about it and get back to you.

Reply:
Alright, fixed this : 

1. Added empty ContainerProperties in here so we can update the types - this won't affect any extensions currently using this since ContainerProperties and ComponentProperties are the same as far as the transpiler is concerned
2. Also fixed up the other container types to use this new ContainerProperties - although we can leave the actual implementation of stuff like aria-live to be done as needed down the road